### PR TITLE
Develop landing page landing page header #121

### PR DESCRIPTION
**Pull request by KyleHefner**
Develop landing page page header.

### CODE REVIEW

**Comment by CSchmitz81** [comment]
For #121

File: src/ctree-app/ctree-landing.html [hunk 1]
```diff
@@ -79,44 +79,70 @@
...
+      #menuButton{
+        color: black;
+        float: right;
       }
       app-drawer ul {
         text-align: left !important;
       }
       app-header-layout {
-        height: 100vh;
+        height: 200vh;
```
Comment (CSchmitz81) [inline src/ctree-app/ctree-landing.html]: Hmm, I'll need to check this out in practice, but it looks strange to essentially be 200% of the view height.  Maybe I'm wrong, but I'll check this out when testing.

**Comment by KyleHefner** [comment]
ok I will work on it again, I spent two hours trying to figure out how to
change the selected text color to black to no avail, but I will try again.

**Comment by KyleHefner** [comment]
I mean green

**Comment by CSchmitz81** [comment]
If you can't figure something out you can just note that the implementation isn't complete and we can note it in the ticket and merge what we have so far.  At least it will be closer to correct and we can continue working to try and resolve any lingering differences.

As far as the specific issue, I would first check if I could update the style by inspecting the CSS in the browser and changing it there (sometimes I need to mark a style as `!important`).  Once I got that, I'd see what style changes it to bold and try to put the color change in the same place.  If you still can't figure it out quickly feel free to leave it and make a note that you need help.  It might also be a good idea to post somewhere online, especially our Slack group, to see if anyone can help.  I'm always asking coworkers what they think about how I'm planning to implement things or suggestions for solving issues, so we should treat the Slack group the same way.

**Comment by CSchmitz81** [comment]
Is this something you're planning to work on during the hackathon this Saturday?

**Comment by CSchmitz81** [comment]
Merging, since progress has stalled, so we have the changes so far.  Work will continue on #121.

**Comment by CSchmitz81** [comment]
This caused regressions, so reversing this merge.

**Pull request merged**